doc/users/interactive.rst

```diff
@@ -146,45 +139,43 @@ Interactive mode controls:
 
 - whether created figures are automatically shown
 - whether changes to artists automatically trigger re-drawing existing figures
-- whether `.pyplot.show` returns immediately or after all of the
-  figures have been closed when given no arguments
-
+- when `.pyplot.show` exits: immediately, or after all of the figures have been closed if given no arguments
 
-If in interactive mode, then:
+If in interactive mode:
 
-- newly created figures will be shown immediately
-- figures will automatically redraw on change
-- pyplot.show will return immediately by default
+- newly created figures will be displayed immediately
+- figures will automatically redraw when elements are changed
+- `pyplot.show` displays the figures and immediately returns to the prompt
```

Comment:
That assumes you're in a prompt...

Though in a script, if `plt.ion()` is called, then `plt.show()` immediately returns, and your script exits because nothing is running, so I guess you do end up at a prompt eventually (just not Python's).

Reply:
Yeah, this is my frustration w/ the plt.show() docs - they kind of assume a prompt too and it's really unclear.

Reply:
Not really? It just says `show` won't return, and nothing about prompts.

Reply:
First line of the show docs:
>When running in ipython with its pylab mode, display all figures and return to the ipython prompt

And the rest of the docs use the language of blocking but don't saw what's being blocked.

Reply:
That is no longer the first line of `show()` docs.

Reply:
🤦